wallet/wallet.go

```diff
@@ -398,39 +399,20 @@ func (w *Wallet) VoteBits() stake.VoteBits {
 	return vb
 }
 
-// AgendaChoice describes a user's choice for a consensus deployment agenda.
-type AgendaChoice struct {
-	AgendaID string
-	ChoiceID string
-}
-
-type AgendaChoices []AgendaChoice
-
-// Map returns the agenda choices formatted as map["AgendaID"] = "ChoiceID".
-func (a AgendaChoices) Map() map[string]string {
-	choices := make(map[string]string, len(a))
-
-	for _, c := range a {
-		choices[c.AgendaID] = c.ChoiceID
-	}
-	return choices
-}
-
 // AgendaChoices returns the choice IDs for every agenda of the supported stake
 // version.  Abstains are included.  Returns choice IDs set for the specified
 // non-nil ticket hash, or the default choice IDs if the ticket hash is nil or
 // there are no choices set for the ticket.
-func (w *Wallet) AgendaChoices(ctx context.Context, ticketHash *chainhash.Hash) (choices AgendaChoices, voteBits uint16, err error) {
+func (w *Wallet) AgendaChoices(ctx context.Context, ticketHash *chainhash.Hash) (choices map[string]string, voteBits uint16, err error) {
```

Comment:
we'll need to bump to v4 before we can start to make breaking api changes again.  v3 was tagged during the 1.8.0 release.

Reply:
Opened a seperate PR for that decred/dcrwallet#2248